### ISSUE TITLE
Multiple manifest.json/catalog.json/dbt_project.yml files found in path ./

### ISSUE THREAD

**Issue by arniwesth** [lightdash/dbt2looker#15]
When running

```
dbt2looker --tag test
```
I get
```
$ dbt2looker --tag test
19:31:20 WARNING Multiple manifest.json files found in path ./ this can lead to unexpected behaviour
19:31:20 WARNING Multiple catalog.json files found in path ./ this can lead to unexpected behaviour
19:31:20 WARNING Multiple dbt_project.yml files found in path ./ this can lead to unexpected behaviour
19:31:20 INFO   Generated 0 lookml views in ./lookml/views
19:31:20 INFO   Generated 1 lookml model in ./lookml
19:31:20 INFO   Success
```
and no lookml files are generated.

I assume this is because I have multiple dbt packages installed? Is there a way to get around this? Otherwise, a feature request would be the ability to specify which files should be used - perhaps in a separate `dbt2looker.yml` settings file.

**Comment by owlas** [comment]
Hey @arniwesth - so this is due to some pretty lazy parsing on my side. 

I'm going to discard the `dbt_project.yml` since it's not needed anymore and that will remove the clash with the dbt packages installed.

Although my understanding was that all the packages were compiled in the top-level manifest.json and catalog.json. Do packages also introduce duplicates of those files on your system?

**Comment by arniwesth** [comment]
In my case it appears to be a single package that might not follow best practices:
```
/dbt_modules/microsoft_ads/docs/manifest.json
/dbt_modules/microsoft_ads_source/docs/manifest.json
/target/manifest.json
```
I can delete it the `docs` folders, but then it would have to be done manually each time these packages are installed.

**Comment by owlas** [comment]
Hey @arniwesth this was fixed in `0.8.0` using the `dbt2looker --target-dir ./target --project-dir .` flags!
